faq/index.md

```diff
@@ -0,0 +1,47 @@
+---
+layout: faq
+title: FAQs
+---
+
+## OBO Foundry Frequently Asked Questions (FAQs)
+
+### Policy
+- <a href="/principles/fp-000-summary.html">What are the OBO Foundry Principles?</a>
```

Comment:
would prefer using markdown style links where possible

Reply:
totally agree but i converted the menu to this page using a few regexps. to rewrite it in md would require either more extensive scripting or tedious editing. or i guess i could try an html-to-md converter, if you feel strongly about it.

Reply:
it's not a big deal but markdown syntax is much more straightfowards and less error-prone.

Reply:
if i had created the page from scratch i would have used md. it didn't seem worth it to me to spend a lot of time to convert the html to md -- but if you would like to do that, please feel free! 🙂